app/messaging/agreement-changed.js

```diff
@@ -24,25 +24,33 @@ function selectExpression (ctx) {
       return calcs[i].expression
     }
   }
-  log('No conditions were matched to provide an expression. Check the coverage of the conditions. Default expression will be used.')
-  return 'userInput * paymentRate'
+  throw new Error('No conditions were matched to provide an expression. Check the coverage of the conditions. Default expression will be used.')
```

Comment:
Throw error now rather than use a default expression. Even for the standards I wasn't convinced it was a good idea to have a default but with the standard set now using this it doesn't work on a practical level.
However, there might be some sense in having a default expression on the standard set (add all individual standard values) than there was for the individual standards. Part of the reason I didn't think the default for the individual standard made much sense was due to the likely (and now known) changes to the input, this probably isn't going to be the case for the standard set. Maybe something to consider adding when more clarity is known.

Reply:
I prefer this approach, as in this case I prefer being explicit about the expression you want to use rather than unintentionally relying on a default that may be inappropriate.

You should remove the `Default expression will be used` from the error message.